lib/workers/popper.js

```diff
@@ -30,7 +28,7 @@ class Popper {
    * Pop a job from the queues.
    *
    * Returns a promise that resolves with the popped job. After this has been
-   * stopped, successive call to `pop` will resolve immediately with `null`.
+   * stopped, successive call to `pop` will resolve immediately with null.
```

Comment:
It's a little odd that you dropped the backticks on this line but added new ones on line 58 below.

Reply:
Whoops -- that was a typo. I had changed the comment when I was messing with the functionality and I meant to revert it to its original. I'll fix that.